rust-webvr/src/api/googlevr/display.rs

```diff
@@ -344,16 +354,21 @@ impl GoogleVRDisplay {
             synced_head_matrix: gvr_identity_matrix(),
             fbo_id: 0,
             fbo_texture: 0,
-            display_id: utils::new_id(),
+            display_id,
             presenting: false,
             paused: false,
             new_events_hint: false,
             pending_events: Mutex::new(Vec::new()),
             processed_events: Mutex::new(Vec::new()),
             attributes: Default::default(),
+            gamepad
         }))
     }
 
+    pub fn gamepad(&self) -> Option<&GoogleVRGamepadPtr> {
```

Comment:
Should this return a `Option<GoogleVRGamepadPtr>`?

Reply:
I don't want to clone when it's not necessary, you don't need the new arc for stop()